### ISSUE TITLE
Size of the test set (annotations.json)

### ISSUE THREAD

**Issue by qishenghu** [abhika-m/FAVA#3]
Thanks for the good work.

According to the arxiv paper, there should be approximately 1,000 annotated response for FAVABENCH detection task. But seems like the 'annotations.json' file from this link (https://huggingface.co/datasets/fava-uw/fava-data/tree/main) contains only 460 records. Could you kindly help me understand which file might be the correct annotated response for FAVABENCH?

Thanks!

**Comment by khunkin** [comment]
The 'annotations.json' file from this [link](https://huggingface.co/datasets/fava-uw/fava-data/tree/main)  should correspond to version 2 of the submission on arXiv, which states: "Our benchmark consists of about 400 responses of ChatGPT and Llama2-Chat 70B." Version 3, submitted on February 21, 2024, mentions: "...annotating approximately 1,000 responses of three widely used LMs." To date (April 18, 2024), the authors have not updated the data to the latest version. Hope that the authors will update the data.

Thanks!

**Comment by khunkin** [comment]
The 'annotations.json' file from this [link](https://huggingface.co/datasets/fava-uw/fava-data/tree/main)(submitted on Jan 15, 2024) should correspond to version 2(submitted on Jan 17, 2024) of the submission on arXiv, which states: "Our benchmark consists of about 400 responses of ChatGPT and Llama2-Chat 70B." And the version 3 (submitted on Feb 21, 2024) mentions: "...annotating approximately 1,000 responses of three widely used LMs." To date (Apr 18, 2024), the authors have not updated the data to the latest version. Hope that the authors will update the data.

Thanks!

**Comment by sunsunshinesunshine** [comment]
I noticed that the provided annotations.json file does not include the evidence documents. Could you please advise on how to obtain the corresponding evidence documents?

**Comment by khunkin** [comment]
> I noticed that the provided annotations.json file does not include the evidence documents. Could you please advise on how to obtain the corresponding evidence documents?

Glad to see this issue is still helping others after 2 years. Regarding your questions:

1. Dataset Reconstruction: Since the official `annotations.json` was incomplete/unusable at the time, I sampled 500 entries from the training set to reconstruct a reliable test set for my experiments.

2. Evidence Documents: Regarding the missing evidence documents, I implemented a **Real-time Fact Retrieval module** using **NER + MediaWiki API**. You can find the detailed implementation and data processing steps in Section 4.1 of [this work](https://arxiv.org/abs/2407.00488).

Hope this helps with your work!

**Comment by sunsunshinesunshine** [comment]
Thank you very much for your detailed and helpful response! Your explanation is extremely helpful for my work. Thanks again for taking the time to respond!
